riscv/csrs.cc

```diff
@@ -1717,3 +1717,12 @@ void srmcfg_csr_t::verify_permissions(insn_t insn, bool write) const {
   if (state->v)
       throw trap_virtual_instruction(insn.bits());
 }
+
+ssp_csr_t::ssp_csr_t(processor_t* const proc, const reg_t addr, const reg_t mask, const reg_t init):
+  masked_csr_t(proc, addr, mask, init) {
+}
+
+void ssp_csr_t::verify_permissions(insn_t insn, bool write) const {
+  DECLARE_XENVCFG_VARS(SSE);
```

Comment:
This needs to call parent `masked_csr_t::verify_permissions()` to do basic CSR privilege enforcement.